database/src/main/postgres/flows/V1.9.1__get_flow_checkpoints_v2.sql

```diff
@@ -14,10 +14,12 @@
  * limitations under the License.
  */
 
-CREATE OR REPLACE FUNCTION flows.get_flow_checkpoints(
-    IN  i_partitioning_of_flow JSONB,
+-- Function: flows.get_flow_checkpoints_v2(4)
+CREATE OR REPLACE FUNCTION flows.get_flow_checkpoints_v2(
```

Comment:
No need to be marked with v2
```suggestion
CREATE OR REPLACE FUNCTION flows.get_flow_checkpoints(
```

It has consequences bellow